ui_framework/src/components/avatar/avatar.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { VISUALIZATION_COLORS } from '../../services/colors/visualization_colors';
+
+const sizeToClassNameMap = {
+  'none': null,
+  's': 'kuiAvatar--s',
+  'm': 'kuiAvatar--m',
+  'l': 'kuiAvatar--l',
+  'xl': 'kuiAvatar--xl',
+};
+
+export const SIZES = Object.keys(sizeToClassNameMap);
+
+export const KuiAvatar = ({
+  image,
+  name,
+  className,
+  size,
+  ...rest,
+}) => {
+  const classes = classNames(
+    'kuiAvatar',
+    sizeToClassNameMap[size],
+    className
+  );
+
+  let optionalBackgroundImage;
+
+  if (image) {
+    optionalBackgroundImage = 'url(' + image + ')';
+  } else {
+    optionalBackgroundImage = 'none';
+  }
+
+  let optionalInitial;
+  if (name && !image) {
+    optionalInitial = (
+      <span aria-hidden="true">{name.substring(0,1)}</span>
+    );
+  }
+
+  const assignedColor = VISUALIZATION_COLORS[Math.floor(name.length % VISUALIZATION_COLORS.length)];
+
+  const avatarStyle = {
+    backgroundImage: optionalBackgroundImage,
```

Comment:
We can change this line to:

```javascript
backgroundImage: image ? 'url(' + image + ')' : 'none',
```

And then we can delete lines 29 through 35.

Reply:
Hey, what is this kind of expression called? I always want to know how to do these but never know the word to look up :)